framework/src/problems/DumpObjectsProblem.C

```diff
@@ -276,8 +294,10 @@ DumpObjectsProblem::dumpGeneratedSyntax(const std::string path)
   if (pathit == _generated_syntax.end())
     return;
 
+  Moose::out << "**START DUMP DATA**\n";
```

Comment:
Is this here because you're parsing the console output in the plugin?

Reply:
Yep